client/src/components/Panel.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import styled from "styled-components";
+
+function Panel({ name, url, description, clone, ownerAvatar }) {
+  return (
+    <PanelContent>
+      <div className="header">
+        <img src={ownerAvatar} alt={name} />
+        <a className="panel-url" href={url} alt="avatar">
+          {name}
+        </a>
+      </div>
+      <div className="content">
+        <h4>
+          <a href={clone}>{name}</a>
+        </h4>
+        <p>{description}</p>
+      </div>
+    </PanelContent>
+  );
+}
+
+const PanelContent = styled.div`
+  width: 45%;
+  display: flex;
+  justify-content: flex-start;
+  box-sizing: border-box;
```

Comment:
You should order all the css alphabetically :)

Reply:
Shhh... We don't want to be in the minority:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/